Makefile

```
@@ -69,22 +101,15 @@ install:
 	$(GO_CMD) install -v $(LDFLAGS) ./cmd/...
 
 image: yamls
-	$(IMAGE_BUILD_CMD) --build-arg VERSION=$(VERSION) \
-	    --target full \
-	    --build-arg HOSTMOUNT_PREFIX=$(CONTAINER_HOSTMOUNT_PREFIX) \
-	    --build-arg BASE_IMAGE_FULL=$(BASE_IMAGE_FULL) \
-	    --build-arg BASE_IMAGE_MINIMAL=$(BASE_IMAGE_MINIMAL) \
-	    -t $(IMAGE_TAG) \
-	    $(foreach tag,$(IMAGE_EXTRA_TAGS),-t $(tag)) \
-	    $(IMAGE_BUILD_EXTRA_OPTS) ./
-	$(IMAGE_BUILD_CMD) --build-arg VERSION=$(VERSION) \
-	    --target minimal \
-	    --build-arg HOSTMOUNT_PREFIX=$(CONTAINER_HOSTMOUNT_PREFIX) \
-	    --build-arg BASE_IMAGE_FULL=$(BASE_IMAGE_FULL) \
-	    --build-arg BASE_IMAGE_MINIMAL=$(BASE_IMAGE_MINIMAL) \
-	    -t $(IMAGE_TAG)-minimal \
-	    $(foreach tag,$(IMAGE_EXTRA_TAGS),-t $(tag)-minimal) \
-	    $(IMAGE_BUILD_EXTRA_OPTS) ./
+	$(IMAGE_BUILD_CMD) $(IMAGE_BUILD_ARGS) $(IMAGE_TAG_FULL)
+	$(IMAGE_BUILD_CMD) $(IMAGE_BUILD_ARGS) $(IMAGE_TAG_MINIMAL)
+
+image-all: OUTPUT=--pull
+image-all: IMAGE_BUILD_CMD=$(IMAGE_BUILDX_CMD)
+image-all: ensure-buildx yamls
+	$(IMAGE_BUILD_CMD) $(IMAGE_BUILD_ARGS) $(IMAGE_TAG_FULL)
```

We could possibly get rid of the `OUTPUT` variable. WDYT?

```suggestion
image-all: IMAGE_BUILD_CMD=$(IMAGE_BUILDX_CMD)
image-all: ensure-buildx yamls
	$(IMAGE_BUILD_CMD) $(IMAGE_BUILD_ARGS) $(IMAGE_TAG_FULL) --pull
```